filebeat/tests/system/test_registrar.py

```diff
@@ -313,6 +313,8 @@ def test_rotating_file_with_shutdown(self):
             lambda: self.output_has(lines=1),
             max_timeout=10)
 
+        time.sleep(1)
+
```

Comment:
Yes, but I couldn't find an easy solution to query the log to exactly contain what I needed, as writing registrar can happen often. This was an easy fix. Will clean up testa after filebeat is cleaned up.